Anishchenko_V/HW5_loops/task2.py

```diff
@@ -0,0 +1,12 @@
+user_number = int(
```

Comment:
good job
look at this implementation:
user_number = int(input('Enter a natural number (i.e. a whole number that is greater than 0): '))

fibonacci_list = [0, 1]

while True:
    next_number = fibonacci_list[-1] + fibonacci_list[-2]
    if next_number > user_number:
        break
    fibonacci_list.append(next_number)

print(f'List of Fibonacci numbers up to your number ({user_number}):')
print(fibonacci_list)

Reply:
Thank you. The alternative implementation is noted, the only my comment here is that for a specific case when user_number = 1 my implementation return the sequence of [0, 1] whilst the alternative one - [0, 1, 1].
I am not sure which outcome for this specific case is correct / as required by the terms of the task.

Reply:
you are right, according to the condition it is more correct for the program to return [0,1]